classes/AddThis.php

```diff
@@ -522,7 +399,7 @@ private function getAttributeUrl($options) {
     return array();
   }
 
-  private function getServiceNamesAsCommaSeparatedString($services) {
+  public function getServiceNamesAsCommaSeparatedString($services) {
```

Comment:
I made this public so it can be used in the Script manager. But is this correct to do?